.github/workflows/systemtests.yml

```diff
@@ -2,7 +2,7 @@ name: System Tests
 
 on:
   push:
-    branches: [ "feature_systemtests" ]
+    branches: [ "feature-systemtests-better" ]
```

Comment:
these two lines can be removed, now that it's in main. You can trigger manually on github (even select any branch)